### ISSUE TITLE
Question : acc: 81.97% but predict result always 0

### ISSUE THREAD

**Issue by x4080** [keras-team/keras#10292]
Please make sure that the boxes below are checked before you submit your issue. If your issue is an implementation question, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [join the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) and ask there instead of filing a GitHub issue.

Thank you!

- [x] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/keras-team/keras.git --upgrade --no-deps

- [x] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [ ] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [ ] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

Hi, I can't figure out what's wrong, I just modified script from pima indian database.csv with my data and
predict result always 0 (but accuracy and training is 81.97%)

data is attached (rename it to data.csv first)

[data.txt](https://github.com/keras-team/keras/files/2042225/data.txt)

Thanks and I'm sorry if I made stupid mistake (very newbie about this ML) :)

```
from keras.models import Sequential
from keras.layers import Dense
import numpy
# fix random seed for reproducibility
numpy.random.seed(7)

dataset = numpy.loadtxt("data.csv", delimiter=",")
# split into input (X) and output (Y) variables
X = dataset[:,0:7]
Y = dataset[:,7]


# create model
model = Sequential()
model.add(Dense(12, input_dim=7, activation='relu'))
model.add(Dense(8, activation='relu'))
model.add(Dense(1, activation='sigmoid'))

# Compile model
model.compile(loss='binary_crossentropy', optimizer='adam', metrics=['accuracy'])

# Fit the model
print('Training')
model.fit(X, Y, epochs=20, batch_size=16,verbose=0)

# evaluate the model
scores = model.evaluate(X, Y)
print("\n%s: %.2f%%" % (model.metrics_names[1], scores[1]*100))

#predictions = model.predict(XTEST)
predictions = model.predict(X)
# round predictions
rounded = [round(x[0]) for x in predictions]
print(rounded)
```

**Comment by JamesTrick** [comment]
@x4080 Not a prob! It was a bit of a sneaky issue!

Best of luck training your model, hope it works out for you :)

**Comment by tRosenflanz** [comment]
`.fit` method has class_weight argument which can help you rebalance the losses of data without changing the data `class_weight={0:1,1:5}` should work

**Comment by x4080** [comment]
Thanks @tRosenflanz and @JamesTrick 

I will close this because I finally found out the problem :)
